agent/consul/authmethod/testing.go

```diff
@@ -0,0 +1,47 @@
+package authmethod
+
+import (
+	"sort"
+
+	"github.com/hashicorp/go-bexpr"
+	"github.com/mitchellh/go-testing-interface"
+	"github.com/stretchr/testify/require"
+)
+
+// RequireIdentityMatch tests to see if the given Identity matches the provided
+// projected vars and filters for testing purpose.
+func RequireIdentityMatch(t testing.T, id *Identity, projectedVars map[string]string, filters ...string) {
+	t.Helper()
+
+	require.Equal(t, projectedVars, id.ProjectedVars)
+
+	names := make([]string, 0, len(projectedVars))
+	for k, _ := range projectedVars {
+		names = append(names, k)
+	}
+	sort.Strings(names)
+	require.Equal(t, names, id.ProjectedVarNames())
+
+	require.Nil(t, id.EnterpriseMeta)
+	// defaultEntMeta := *structs.DefaultEnterpriseMeta()
```

Comment:
Do we typically just comment out ENT stuff versus erasing it? I guess it makes it easier to read maybe? Just checking this is meant to be here. 🤔

Reply:
ha. Actually that's dead code. I fixed a bug elsewhere and forgot to clean up.